src/CreeDictionary/CreeDictionary/templatetags/relabelling.py

```diff
@@ -19,11 +19,14 @@
 # If a paradigm label preference is not set, use this one!
 DEFAULT_PARADIGM_LABEL = "english"
 
-label_setting_to_relabeller = {
-    "english": LABELS.english,
-    "linguistic": LABELS.linguistic_short,
-    "nehiyawewin": LABELS.cree,
-}
+
+def label_setting_to_relabeller():
+    labels = read_labels()
+    return {
+        "english": labels.english,
+        "linguistic": labels.linguistic_short,
+        "nehiyawewin": labels.cree,
+    }
```

Comment:
Nitpick, but I think it's better to pass in the label setting rather than return a dictionary that will be instantly accessed exactly once, i.e.,

```suggestion
def label_setting_to_relabeller(label_setting: str):
    labels = read_labels()
    return {
        "english": labels.english,
        "linguistic": labels.linguistic_short,
        "nehiyawewin": labels.cree,
    }[label_setting]
```

[...]